src/components/WhaleSightingViewer/WhaleSightingViewer.tsx

```diff
@@ -1,6 +1,10 @@
+import { Login } from "../Login/Login";
 import "./WhaleSightingViewer.scss";
 
-export function WhaleSightingViewer() {
+export function WhaleSightingViewer(props: {loggedIn:boolean}) {
```

Comment:
Can we use an interface for the props? Then when you use the property you can just access using `loggedIn`
```suggestion
interface WhaleSightingViewerProps {
    loggedIn: boolean;
}

export function WhaleSightingViewer({loggedIn:boolean}: WhaleSightingViewerProps) {
```